src/OpenTelemetry.Extensions.Hosting/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.4.0-rc.1
```

Comment:
OpenTelemetry.Extensions.Hosting to now align with the core packages release versions.